src/modules/dashboard/components/Dialogs/CancelStreamingDialog/FundingSourceItem.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useCallback, useMemo } from 'react';
+import classNames from 'classnames';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+
+import ColorTag, { Color } from '~core/ColorTag';
+import { InputLabel } from '~core/Fields';
+import Numeral from '~core/Numeral';
+import { AnyToken, Colony } from '~data/index';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+
+import styles from './CancelStreamingDialog.css';
+
+const MSG = defineMessages({
+  fundingSource: {
+    id: `dashboard.CancelStreamingDialog.CancelStreamingForm.fundingSource`,
+    defaultMessage: 'Funding Source',
+  },
+  team: {
+    id: `dashboard.CancelStreamingDialog.CancelStreamingForm.team`,
+    defaultMessage: 'Team',
+  },
+  rate: {
+    id: `dashboard.CancelStreamingDialog.CancelStreamingForm.rate`,
+    defaultMessage: 'Rate',
+  },
+  limit: {
+    id: `dashboard.CancelStreamingDialog.CancelStreamingForm.limit`,
+    defaultMessage: 'Limit',
+  },
+});
+
+interface Props {
+  colony: Colony;
+  filteredDomainId: string;
+  index: number;
+  isMultiple?: boolean;
+  rateTime: string;
+  rateAmount: number;
+  rateToken: AnyToken;
+}
+
+const displayName = 'dashboard.CancelStreamingDialog.FundingSourceItem';
+
+const FundingSourceItem = ({
+  colony,
+  rateToken,
+  rateAmount,
+  rateTime,
+  filteredDomainId,
+  index,
+  isMultiple,
+}: Props) => {
+  const domain = useMemo(
+    () =>
+      colony?.domains.find(
+        ({ ethDomainId }) => Number(filteredDomainId) === ethDomainId,
+      ),
+    [colony, filteredDomainId],
+  );
+
+  const getDomainColor = useCallback<(domainId: string | undefined) => Color>(
+    (domainId) => {
+      const rootDomainColor: Color = Color.LightPink;
+      const defaultColor: Color = Color.Yellow;
+      if (domainId === String(ROOT_DOMAIN_ID)) {
+        return rootDomainColor;
+      }
+      if (!colony || !domainId) {
+        return defaultColor;
+      }
+      return domain ? domain.color : defaultColor;
+    },
+    [colony, domain],
+  );
+
+  const fundingSourceTitle = isMultiple ? `${index + 1}: ` : '';
```

Comment:
There should be a period (.) and not a colon ( : ) after the number as per Figma designs:
<img width="371" alt="Screen Shot 2022-10-03 at 6 46 21 PM" src="https://user-images.githubusercontent.com/71563622/193633231-c96f44a1-d2c8-4ea1-acb1-d5d2ffb171b6.png">

Reply:
it is changed